auth0/v3/authentication/database.py

```diff
@@ -32,8 +32,7 @@ def login(self, client_id, username, password, connection, id_token=None,
                 'device': device,
                 'grant_type': grant_type,
                 'scope': scope,
-            },
-            headers={'Content-Type': 'application/json'}
```

Comment:
All these args were removed because they match the default behavior https://github.com/auth0/auth0-python/blob/master/auth0/v3/authentication/base.py#L24